packages/cli-generator/build.ts

```diff
@@ -0,0 +1,100 @@
+import { readFileSync, writeFileSync } from 'node:fs';
+import { execSync } from 'node:child_process';
+import { slug } from 'github-slugger';
+
+interface Command {
+  name: string;
+  description: string;
+}
+
+function getSubcommands(commandOutput: string): Command[] {
+  const subcommands = [];
+
+  let readingSubcommands = false;
+  for (const line of commandOutput.split('\n').map((l) => l.trim())) {
+    if (readingSubcommands) {
+      if (line.length === 0) {
+        readingSubcommands = false;
+      } else {
+        const subcommand = line.substring(0, line.indexOf(' '));
+        const description = line.substring(subcommand.length).trim();
+        if (subcommand !== 'help') {
+          subcommands.push({ name: subcommand, description });
+        }
+      }
+    }
+    if (line === 'Commands:') {
+      readingSubcommands = true;
+    }
+  }
+
+  return subcommands;
+}
+
+function generateCommandDoc(command: string, level: number, subcommandList: Command[]): string {
+  const output = execSync(`pnpm tauri ${command} --help`)
```

Comment:
This approach is kinda fragile whenever we change the layout of the help output but I don't think that will happen more than once.
Also running the binary just for the help output sounds weird when we have access to the docs/source code itself since we have tauri as a git submodule.

Is tauri auto generating the rustdocs/scheme and we can consume them the same way we do for the permissions and configs? If not is it not feasible to do so?